package-lock.json

```diff
@@ -6039,7 +6039,6 @@
       "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.2.tgz",
       "integrity": "sha512-iownA+hC4uHFp+7gwP/y5SzaiUo7m2vpa0dhpzw8YuKtiZsz7cIXsFbXpLEeBM6WuCQyw1MH4RRe6XI8GFUctQ==",
       "dev": true,
-      "optional": true,
```

Comment:
I think the removal of this will explode things -- @efischer19 ran into this, this week.

Reply:
https://media.giphy.com/media/MVUyVpyjakkRW/giphy.gif

Reply:
oh good catch. My docker container is being very insistent about removing these lines for some reason...

Reply:
Is it possible our node version is getting updated unexpectedly? 8.1.11 came out less than a month ago according to https://nodejs.org/en/download/releases/

Reply:
Hmm. Maybe I did accidentally run `npm install` on my host. Cause, I just ran `git checkout origin/master -- package-lock.json` and then `npm install` again and it's not removing the optional line anymore.